Plugins/Traccc/include/Acts/Plugins/Traccc/Detail/Module.hpp

```diff
@@ -0,0 +1,84 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+// Acts include(s)
+#include "Acts/Geometry/GeometryHierarchyMap.hpp"
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+
+// Detray include(s)
+#include "detray/core/detector.hpp"
+
+// Traccc include(s)
+#include "traccc/edm/cell.hpp"
+#include "traccc/geometry/geometry.hpp"
+#include "traccc/io/digitization_config.hpp"
+#include "traccc/io/read_geometry.hpp"
+#include "traccc/io/reader_edm.hpp"
+
+// System include(s)
+#include <cstdint>
+#include <cstdlib>
+#include <map>
+#include <memory>
+#include <tuple>
+#include <utility>
+
+namespace Acts::TracccPlugin::Detail {
+
+/// Helper function which finds module from csv::cell in the geometry and
+/// digitization config, and initializes the modules limits with the cell's
+/// properties
+traccc::cell_module get_module(const std::uint64_t geometryID,
```

Comment:
Consider following the ACTS naming convention for these. Is `geometryId` the `Acts::GeometryIdentifier` value? If so, there's a typedef in there you can use.